src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -1282,6 +1282,10 @@ class NodeConfig(object):
             for child in root:
                 if child.tag == "gpu":
                     bus_id = child.get('id')
+                    # make sure the PCI domain is 16 bits and 4 hex numbers
+                    pci_domain, pci_rest = child.get('id').split(":", 1)
+                    fixed_pci_domain = pci_domain[-4:]
+                    bus_id = fixed_pci_domain + ":" + pci_rest
```

Comment:
@altair4 since bus_id is being set here then line 1284 is not needed right?

Reply:
Right - it's now been removed.

Reply:
Please push your commit so we can see the change.